benchmarks/triton_kernels_benchmark/gemm_postop_addmatrix_benchmark.py

```diff
@@ -209,31 +224,31 @@ def matmul(a, b, d, c):
 @benchmark_suit.perf_report(
     benchmark_suit.Benchmark(
         # argument names to use as an x-axis for the plot
-        x_names=['B', 'M', 'K', 'N'],
+        x_names=['B', 'M', 'K', 'N', 'dtype'],
         # different possible values for `x_name`
-        x_vals=[[1, 1024 * i, 1024 * i, 1024 * i] for i in [1, 2, 4, 8]] +  #
-        [  #
-            [1, 1, 5120, 13824],  #
-            [1, 4, 4096, 12288],  #
-            [1, 512, 8192, 8192],  #
-            [1, 512, 8192, 32768],  #
-            [1, 512, 32768, 8192],  #
-            [1, 1024, 16384, 8192],  #
-            [1, 1024, 28672, 8192],  #
-            [1, 3072, 4096, 3072],  # FIXME: Remove this case when gemm_streamk_benchmark works
-            [1, 4096, 16384, 8192],  #
-            [1, 8192, 16384, 1024],  #
-            [1, 8192, 16384, 4096],  #
-            [1, 16384, 1024, 8192],  #
-            [1, 16384, 4096, 8192],  #
-            [1, 16384, 8192, 1024],  #
-            [1, 16384, 8192, 4096],  #
-            [4, 32768, 128, 4096],  #
-            [4, 32768, 4096, 128],  #
-            [32, 4096, 4096, 128],  #
-            [4096, 8, 128, 16384],  #
-            [4096, 8, 16384, 128]
-        ],
+        x_vals=[[1, 1024 * i, 1024 * i, 1024 * i, dtype] for i in [1, 2, 4, 8] for dtype in dtypes()] +  #
+        [[*shape, dtype]
+         for shape in [[1, 1, 5120, 13824],  #
+                       [1, 4, 4096, 12288],  #
+                       [1, 512, 8192, 8192],  #
+                       [1, 512, 8192, 32768],  #
+                       [1, 512, 32768, 8192],  #
+                       [1, 1024, 16384, 8192],  #
+                       [1, 1024, 28672, 8192],  #
+                       [1, 3072, 4096, 3072],  # FIXME: Remove this case when gemm_streamk_benchmark works
+                       [1, 4096, 16384, 8192],  #
+                       [1, 8192, 16384, 1024],  #
+                       [1, 8192, 16384, 4096],  #
+                       [1, 16384, 1024, 8192],  #
+                       [1, 16384, 4096, 8192],  #
+                       [1, 16384, 8192, 1024],  #
+                       [1, 16384, 8192, 4096],  #
+                       [4, 32768, 128, 4096],  #
+                       [4, 32768, 4096, 128],  #
+                       [32, 4096, 4096, 128],  #
+                       [4096, 8, 128, 16384],  #
+                       [4096, 8, 16384, 128]]
```

Comment:
[nit] easier to command out shapes when debugging
```suggestion
                       [4096, 8, 16384, 128] #
                       ]
```